bestofjs/bestofjs-webui
# Format dates with browser native `Intl.DateTimeFormat`?

## Goal

Use browser's native `Intl.DateTimeFormat` API to format dates instead of using `date-fns`s `format` function, following the approach described here: https://dev.to/dmtrkovalenko/you-might-not-need-date-fns-23f7

This way we could remove `date-fns` dependency.

This is an experiment as I have never used this browser API before.

## How to test

Check the dates at 2 following places:

- At the footer of all page, the time data was updated: `The last update was 3 hours ago (at 6:16 AM).`
- In the `/timeline` page, check the dates of creation for each project (E.g. `August 2006`)

## Screenshots

### Footer

![image](https://user-images.githubusercontent.com/5546996/169674450-aa9cd8f7-dd9e-405d-835d-d6522bdaf632.png)

### Timeline

![image](https://user-images.githubusercontent.com/5546996/169674459-718b6061-0d5b-4c26-aeba-124c2a0d9ffb.png)
